pkg/mkimage-raw-efi/config.json

```diff
@@ -217,10 +217,20 @@
                 "rbind"
             ]
         },
+        {
+            "destination": "/opt/interactive-installer",
+            "type": "bind",
+            "source": "/containers/onboot/002-interactive-installer/tmp/upper",
```

Comment:
Why tmp/upper and not lower like the other bind mounts?

Reply:
First, I tried using "lower" as in the other bind mounts, but it did not work. I also need to figure out why. Maybe @deitch has an idea.

Reply:
This thing really is ugly, isn't it? All of these explicit mounts into specific paths? What is the purpose of mounting it?

> but it did not work. I also need to figure out why

Can you describe what it does, what you expected, and how it failed?

Reply:
I need to access installer.json from the pkg/mkimage-raw-efi container to expose the parameters the user inserted. I searched for the file ("installer.json"), and it is not in `/containers/onboot/002-interactive-installer/lower`, as I expected, maybe because it is an onboot container, not a service one!